build_library/prod_image_util.sh

```diff
@@ -102,7 +103,13 @@ create_prod_image() {
   fi
 
   tar -cf "${BUILD_DIR}/${image_pkgdb}" -C "${root_fs_dir}" var/cache/edb var/db/pkg
-
+  sudo cp -a "${root_fs_dir}" "${BUILD_DIR}/root_fs_dir2"
+  sudo rsync -a --delete  "${BUILD_DIR}/configroot/etc/portage" "${BUILD_DIR}/root_fs_dir2/etc"
+  sudo mksquashfs "${BUILD_DIR}/root_fs_dir2"  "${BUILD_DIR}/${image_sysext_base}" -noappend
+  sudo rm -rf "${BUILD_DIR}/root_fs_dir2"
+  # Run build_sysext script to create python sysext so we can see it from GitHub actions CI workflow; TO REMOVE
+  sudo "$(dirname ${BASH_SOURCE[0]})/../build_sysext" --board="${BOARD}" --group="${FLAGS_group}" --output_root="${FLAGS_output_root}" --build_attempt="${FLAGS_build_attempt}" --squashfs_base="${BUILD_DIR}/${image_sysext_base}" python-sysext python
```

Comment:
Before merge we should delete this and also squash the commits and add a longer commit message (the body should have two paragraphs, one describing the problem/goal, one describing what the changes do)